src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -43,12 +47,14 @@ function HeatmapVis(props: Props): JSX.Element {
         orientation="bottom"
         numberPixels={dims[1]}
       />
-      {domain && (
-        <ColorBar
-          className={styles.rightArea}
-          interpolator={INTERPOLATORS[colorMap]}
-          dataBounds={domain}
-        />
+      {colorScale && (
+        <div className={styles.rightArea}>
+          <LogScaleToggler logScale={logScale} setLogScale={setLogScale} />
+          <ColorBar
+            interpolator={INTERPOLATORS[colorMap]}
+            colorScale={colorScale}
```

Comment:
`colorScale` and `interpolator` are always passed as props together. It might be worthwhile to make an object out of both.

Reply:
The common state will help with that.